tests/rptest/tests/cluster_config_test.py

```diff
@@ -1418,24 +1450,24 @@ def test_aliasing_with_upgrade(self, wipe_cache: bool):
                            than just cache.
         """
 
-        old_version, _ = self.installer.latest_for_line(self.legacy_version)
+        old_version = self.installer.highest_from_prior_feature_version(
+            prop_set.redpanda_version)
         self.installer.install(self.redpanda.nodes, old_version)
 
         self.redpanda.start()
 
         # Check we're running a version where the alias name is actually the primary
         # (i.e. older than when the alias was introduced)
         cluster_config = self.admin.get_cluster_config(include_defaults=True)
-        assert self.primary_name not in cluster_config
-        assert self.aliased_name in cluster_config
-
-        value_old_version = 1230
+        assert prop_set.primary_name not in cluster_config
+        assert prop_set.aliased_name in cluster_config
 
         self.redpanda.set_cluster_config(
-            {self.aliased_name: value_old_version})
-        self._check_value_everywhere(self.aliased_name, value_old_version)
+            {prop_set.aliased_name: prop_set.test_values[0]})
+        self._check_value_everywhere(prop_set.aliased_name,
+                                     prop_set.test_values[0])
 
-        self.installer.install(self.redpanda.nodes, RedpandaInstaller.HEAD)
```

Comment:
Curious, I'm surprised we were getting by with this upgrade from 23.1 to HEAD

Reply:
me too. upgrade tests are kinda in a broken state right now, i'm adding a ticket for later